config.yaml

```diff
@@ -0,0 +1,72 @@
+admin:
```

Comment:
This file will be removed before the final review. It's only included here to demonstrate how I tested.